App.jsx

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-import { useSelector } from 'react-redux'
-import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack'
-import HomeScreen from './components/HomeScreen'
+import React from "react";
+import { useSelector } from "react-redux";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import HomeScreen from "./components/HomeScreen";
+import SingleArticle from "./components/SingleArticle";
 
-
-const Stack = createStackNavigator()
+const Stack = createStackNavigator();
 
 const App = () => {
-  const { appHeader } = useSelector(state => state)
+  const { appHeader } = useSelector((state) => state);
```

Comment:
Please remove the parenthesis around (state) for readability

Reply:
fixed :)